kodilla-stream/src/main/java/com/kodilla/stream/array/ArrayOperations.java

```diff
@@ -0,0 +1,24 @@
+package com.kodilla.stream.array;
+
+import java.util.OptionalDouble;
+import java.util.stream.IntStream;
+
+public interface ArrayOperations {
+     static double getAverage(int[] numbers){
+
+               System.out.println("Table of int numbers: ");
+               IntStream.range(0,numbers.length)
+                       .map(n->n=numbers[n])
+                       .forEach(System.out::println);
+
+               OptionalDouble result =  IntStream.range(0, numbers.length)
+                       .map(n->n=numbers[n])
+                       .average();
+
+               if (result.isPresent()){
+                    return result.getAsDouble();
```

Comment:
Mozna zapisac to prosciej
```
 double result =  IntStream.range(0, numbers.length)
                       .map(n->n=numbers[n])
                       .average()
                       .orElse(0.0)
```

Reply:
:) nie mogłem tego znaleźć jak zapisać prościej... Przetestuję... Dziękuję